clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/api/KubernetesClientApiAdapter.groovy

```diff
@@ -144,4 +144,40 @@ class KubernetesClientApiAdapter {
       return list.items
     }
   }
+
+  V1beta1StatefulSet createStatfulSet(String namespace, V1beta1StatefulSet statefulSet) {
+    exceptionWrapper("StatefulSet.create", "Create Stateful Set ${statefulSet?.metadata?.name}", namespace) {
```

Comment:
The labeling here isn't consistent. Typically the resource's first char is lowecase and the name is pluralized.